neuromation/api/utils.py

```diff
@@ -29,6 +28,8 @@
 else:
     from async_generator import asynccontextmanager
 
+asynccontextmanager
```

Comment:
Is the line required?

Reply:
Yes, it silences warning "imported but not used". `asynccontextmanager` is imported from `utils` in several other places, so I just add a workaround. In different PR I'll propose better solution.

Will add a comment.